security/crunch/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+name            crunch
+version         3.6
+revision        0
+categories      security
+maintainers     {@i0ntempest me.com:szf1234} openmaintainer
+platforms       darwin
+license         GPL-2
+
+description     Wordlist generator
+long_description \
+                Crunch is a wordlist generator where you can specify a standard character \
+                set or a character set you specify. crunch can generate all possible \
+                combinations and permutations.
+
+homepage        https://sourceforge.net/projects/crunch-wordlist/
+master_sites    https://downloads.sourceforge.net/project/crunch-wordlist/crunch-wordlist/
+distname        crunch-${version}
+extract.suffix  .tgz
+patchfiles      patch-destroot-fix.diff
+configure.cmd   true
```

Comment:
Given no configure file, you will need to make sure you are [using the right compiler] to build it. (https://trac.macports.org/wiki/UsingTheRightCompiler)